drivers/utilities/utilities.go

```diff
@@ -0,0 +1,193 @@
+	appTypeAnnotationKey  = "appType"
+)
+
+// RandomString generates a random lowercase string of length characters.
+func RandomString(length int) string {
+	const letters = "abcdefghijklmnopqrstuvwxyz"
+	randomBytes := make([]byte, length)
+	for i := range randomBytes {
+		randomBytes[i] = letters[rand.Intn(len(letters))]
+	}
+	randomString := string(randomBytes)
+	return randomString
+}
+
+// GenerateRandomSQLCommands generates pairs of INSERT, UPDATE, SELECT and DELETE queries for a database
+func GenerateRandomSQLCommands(count int, appType string) map[string][]string {
+	var randomSqlCommands = make(map[string][]string)
+	var tableName string
+	var insertCommands []string
+	var selectCommands []string
+	var deleteCommands []string
+	var updateCommands []string
+
+	if appType == Postgres {
+		tableName = "pg_validation_" + RandomString(5)
+		insertCommands = append(insertCommands, fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
+		key varchar(45) NOT NULL,
+		value varchar(45) NOT NULL
+	  )`, tableName))
+	} else if appType == MySql {
+		tableName = "mysql_validation_" + RandomString(5)
+		insertCommands = append(insertCommands, fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
+			`+"`key` "+`VARCHAR(45) NOT NULL ,
+			value VARCHAR(255)
+		  )`, tableName))
+	}
+
+	for counter := 0; counter < count; counter++ {
+		currentCounter := strconv.Itoa(counter)
+		randomValue := "Value-" + RandomString(10)
+		updatedRandomValue := "Value-Updated-" + RandomString(10)
+		insertCommands = append(insertCommands, fmt.Sprintf("INSERT INTO %s VALUES('%s', '%s')", tableName, currentCounter, randomValue))
+		if appType == Postgres {
+			selectCommands = append(selectCommands, fmt.Sprintf("SELECT * FROM %s WHERE key='%s'", tableName, currentCounter))
+			updateCommands = append(updateCommands, fmt.Sprintf("UPDATE %s SET value='%s' WHERE key='%s'", tableName, updatedRandomValue, currentCounter))
+			deleteCommands = append(deleteCommands, fmt.Sprintf("DELETE FROM %s WHERE key='%s'", tableName, currentCounter))
+		} else if appType == MySql {
+			selectCommands = append(selectCommands, fmt.Sprintf("SELECT * FROM %s WHERE `key`='%s'", tableName, currentCounter))
+			updateCommands = append(updateCommands, fmt.Sprintf("UPDATE %s SET value='%s' WHERE `key`='%s'", tableName, updatedRandomValue, currentCounter))
+			deleteCommands = append(deleteCommands, fmt.Sprintf("DELETE FROM %s WHERE `key`='%s'", tableName, currentCounter))
+		}
+
+	}
+
+	randomSqlCommands["insert"] = insertCommands
+	randomSqlCommands["select"] = selectCommands
+	randomSqlCommands["update"] = updateCommands
+	randomSqlCommands["delete"] = deleteCommands
+
+	// log.Infof("Insert Queries - [%v]", insertCommands)
+	// log.Infof("Select Queries - [%v]", selectCommands)
+	// log.Infof("Update Queries - [%v]", updateCommands)
+	// log.Infof("Delete Queries - [%v]", deleteCommands)
+
+	return randomSqlCommands
+
+}
+
+// GenerateRandomSQLCommands generates pairs of INSERT and SELECT queries for a database
```

Comment:
Fix the doc string to have the name of the function

Reply:
Done